test/test_utils/test_path_utils.py

```diff
@@ -0,0 +1,58 @@
+import pathlib
+import pytest
+
+from corppa.utils.path_utils import encode_htid, decode_htid
+from corppa.utils.path_utils import get_stub_dir, get_vol_dir
```

Comment:
preferred python style is to group all imports from the same module
```suggestion
from corppa.utils.path_utils import encode_htid, decode_htid, get_stub_dir, get_vol_dir
```

Also, I just discovered that our `ruff` pre-commit hook is not fixing imports for us. (In other projects we've used black and isort together, and I knew that ruff could cover both but didn't know we had to opt into it.) Sound ok to you if we turn this on?

Reply:
Sounds fine with me, I had split these into two lines due to line length; but I think `ruff` is more lenient with line length (allows for a bit over 80)